salt/kubelet/status.sls

```diff
@@ -0,0 +1,5 @@
+{% set localhost = salt['grains.get']('localhost', '') %}
+
+salt://kubelet/check-status.sh:
```

Comment:
Is this valid?

Reply:
Yes, it is valid

Reply:
Maybe it is valid, but I think does not follow the implicit code conventions we have been following for `id`s.  I would just use:

```SaltStack
check-status:
  cmd.script:
    - name: salt://kubelet/check-status.sh
    - args: {{ localhost }}
```